include/SFML/Graphics/Texture.hpp

```diff
@@ -119,17 +110,16 @@ class SFML_GRAPHICS_API Texture : GlResource
     /// The maximum size for a texture depends on the graphics
     /// driver and can be retrieved with the getMaximumSize function.
     ///
-    /// If this function fails, the texture is left unchanged.
-    ///
     /// \param filename Path of the image file to load
     /// \param area     Area of the image to load
     ///
-    /// \return True if loading was successful
+    /// \return Texture if loading was successful, otherwise `std::nullopt`
```

Comment:
```suggestion
    /// \return Texture if loading was successful, `std::nullopt` otherwise
```

Reply:
What makes you feel strongly about this order of words? There are lots of SFML APIs that use slightly different variations of this same sentence so there is no one standard phrasing for me to pick from. Because there is little consistency I don't put too much thought into it. Perhaps it would be worthwhile to go through all our docs and unify our phrasing for APIs with the same semantics.

Reply:
I don't feel strongly about it, it just sounds better in my mind. Feel free to ignore and I agree with your last sentence